Unstructured-IO/pipeline-invoices
# feat: add logging config

Adds a proper `logger_config.yaml` for unicorn.

**Test Instructions**
`make run-web-app` should include formatting output specific to `logger_config.yaml` as shown below.
```
(unstructured) ygyoon@Nathans-MacBook-Pro pipeline-invoices % make run-web-app
PYTHONPATH=/Users/ygyoon/pipeline-invoices uvicorn prepline_invoices.api.app:app --log-config logger_config.yaml --reload
2023-03-31 14:24:55,464 uvicorn.error INFO Will watch for changes in these directories: ['/Users/ygyoon/pipeline-invoices']
2023-03-31 14:24:55,465 uvicorn.error INFO Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
2023-03-31 14:24:55,465 uvicorn.error INFO Started reloader process [98253] using WatchFiles
2023-03-31 14:25:00,724 uvicorn.error INFO Started server process [98264]
2023-03-31 14:25:00,724 uvicorn.error INFO Waiting for application startup.
2023-03-31 14:25:00,725 uvicorn.error INFO Application startup complete.
```
Test hitting the API with:
```
curl -X 'POST' \
  'http://localhost:8000/invoices/v0.1.0/invoices' \
  -H 'accept: application/json' \
  -H 'Content-Type: multipart/form-data' \
  -F 'files=@sample-docs/000d40dd-2812.png' | jq -C . | less -R
```
And note the output line from the `pipeline-invoices` logger:
```
...
2023-03-31 14:40:46,858 127.0.0.1:59632 POST /invoices/v0.1.0/invoices HTTP/1.1 - 200 OK
...
```
